### PR TITLE
[mds-provider] More updates to event processing

### PR DESCRIPTION
**Pull request by mdurling**
All the provider event processing is still inactive, but wanted to get some of this reviewed/merged and kept current with recent changes to master.

## Impacts
- [x] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance

### CODE REVIEW

**Comment by avatarneil** [comment]
Hey Jenkins! Please run tests <3

File: packages/mds-provider/event-processor/processors/status-changes-processor.ts [hunk 1]
```diff
@@ -66,19 +68,18 @@ function asStatusChange(entry: StatusChangesProcessorEntry): StatusChange {
...
-    recorded: now()
+    recorded,
+    sequence
   }
 }
 
-const isVehicleEventEntry = <Label>(
-  entry: LabeledStreamEntry<Label>
-): entry is LabeledStreamEntry<Label, VehicleEvent, 'event'> =>
+const isVehicleEventEntry = (
```
Comment (avatarneil) [inline packages/mds-provider/event-processor/processors/status-changes-processor.ts]: I think a better name for this would be `isVehicleEventStreamEntry`

Comment (mdurling) [reply packages/mds-provider/event-processor/processors/status-changes-processor.ts]: It's now called `isStatusChangesProcessorStreamEntry` .. I simplified the type-guarding.

**Comment by avatarneil** [comment]
Heyo Jenkins, please rerun tests

**Comment by avatarneil** [comment]
Jenkins my friend, please rerun tests and actually work this time!

**Pull request merged**